java/src/jmri/LightManager.java

```diff
@@ -114,6 +114,7 @@
 
     /**
      * Test if parameter is a properly formatted system name.
+     * Logging should not be higher than WARN to keep silent when used for in line validation.
```

Comment:
This suggests that WARN messages will be suppressed at certain times. We should *never* suppress a warning that is generated from within our own code.

Reply:
Changed comment, removed log Level manipulation